Sources/Penny/Penny.swift

```diff
@@ -11,10 +11,13 @@ struct Penny {
     static func start(mainService: any MainService) async throws {
         /// Use `1` instead of `System.coreCount`.
         /// This is preferred for apps that primarily use structured concurrency.
-        let eventLoopGroup = MultiThreadedEventLoopGroup(numberOfThreads: 1)
         let httpClient = HTTPClient(
-            eventLoopGroupProvider: .shared(eventLoopGroup),
-            configuration: .init(decompression: .enabled(limit: .size(1 << 32)))
+            eventLoopGroup: MultiThreadedEventLoopGroup.singleton,
```

Comment:
`HTTPClient`'s `.init()` does automatically use a singleton but it's definition is different from the ones used in NIO and sometimes choses to use the NIOTS ELG. That's why i've preferred to explicitly declare the ELG used by the `HTTPClient`.

Reply:
@gwynne FYI